### DISCUSSION TITLE
Duplicate an existing alert rule as a starting point for a new alert

### DISCUSSION THREAD

**Discussion by wardbekker** [grafana/grafana#58625] [Alerting]
To increase the productivity when adding new alerts from the UI make it possible to e.g. duplicate an alert rule as a starting point for a new alert. Especially useful if there is a standard of included summary, annotations and custom labels.

**Comment by konrad147** [comment]
This feature was added in Grafana v9.4
https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-4/#copy-alert-rules-and-notification-templates
